### DISCUSSION TITLE
"How can I find materials to learn SO on my own?"

### DISCUSSION THREAD

**Discussion by Carlos-mb** [Security-Onion-Solutions/securityonion#13449] [2.4]
### Version

2.4.100

### Installation Method

Security Onion ISO image

### Description

other (please provide detail below)

### Installation Type

Standalone

### Location

on-prem with Internet access

### Hardware Specs

Exceeds minimum requirements

### CPU

6

### RAM

64

### Storage for /

300

### Storage for /nsm

300

### Network Traffic Collection

span port

### Network Traffic Speeds

Less than 1Gbps

### Status

Yes, all services on all nodes are running OK

### Salt Status

No, there are no failures

### Logs

No, there are no additional clues

### Detail

Hi,

I'm trying to learn Security Onion (SO) on my own and I'm finding it quite difficult to locate good resources.
I've set up a distributed configuration at home using multiple virtual machines and a dedicated server with sufficient resources. I'm working hard to understand all the SO components, but I'm struggling to find suitable learning materials.
I've completed the free training, but I realize I need more comprehensive instruction to become a Security Onion professional.
Any assistance you can provide would be greatly appreciated.

Thanks,
Carlos

### Guidelines

- [X] I have read the discussion guidelines at https://github.com/Security-Onion-Solutions/securityonion/discussions/1720 and assert that I have followed the guidelines.

**Comment by dougburks** [comment]
Here are some additional resources:

Security Onion Documentation
https://securityonion.net/docs

Security Onion Documentation in printed book format:
https://securityonion.com/book

Paid training (on-demand and live instructor-led):
https://securityonion.com/training

Hope that helps!

**Comment by Carlos-mb** [comment]
Thank you. 

I've been using this documentation, but it's insufficient for learning to use SO. For example, the explanations of HUNT are very brief, just a few lines, and there's hardly any information about Kibana. The documentation does link to the Kibana page, but it's not easy to find out how to use it in SO.

I would like to find more extensive documentation to learn SO in depth.

Thank you

**Reply by dougburks** [reply]
The Hunt documentation page at https://docs.securityonion.net/en/2.4/hunt.html is intentionally short because the Hunt and Dashboards interfaces are very similar. Instead of duplicating information between the Hunt documentation and the Dashboards documentation, the Hunt page simply explains the differences between Hunt and Dashboards and then refers you to the Dashboards documentation at https://docs.securityonion.net/en/2.4/dashboards.html for the rest.

The Dashboards documentation at https://docs.securityonion.net/en/2.4/dashboards.html covers Dashboards and most of it applies to Hunt as well. That page also includes a link to our Dashboards video at https://youtu.be/xUBhyF7se8s which might be another helpful resource for you.

While you're on our YouTube channel, you might want to check out the other videos there as they may help you as well.

Do you have a specific question about Hunt, Dashboards, or Kibana that is not covered in the documentation or the YouTube channel?

**Reply by Carlos-mb** [reply]
I thank you very much the effort to explain it to me. 
I'll follow your recommendations. 

Best Regards,
Carlos.
